src/components/SearchForm.js

```diff
@@ -1,10 +1,120 @@
 import React, { useState } from "react";
+import { withFormik, Form, Field, ErrorMessage } from "formik";
+import * as Yup from "yup";
+import axios from "axios";
+import styled from "styled-components";
+
+const Forms = styled.form`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const Lead = styled.h1`
+color: red;
+text-align: center;
+font-weight: bold;
+font-size: 30px`
+
+const Detail = styled.h3`
+margin-bottom: 20px;`
+
+const Enter = styled.button`
+margin-top: 20px;
+width: 70px;
+height: 40px;
+`
```

Comment:
well done on styling
I personally find it more beneficial to create one styled tag and nest others tags inside.